src/mahesh/FirstTest.java

```diff
@@ -0,0 +1,48 @@
+package mahesh;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class FirstTest {
+	public static void main(String[] args) {
+		System.setProperty("webdriver.chrome.driver", ".//chromedriver.exe");
+		WebDriver driver = new ChromeDriver();
+		driver.get("file:///D:/Technocredit/Project/TechnoGitProject/SeleniumBasics/SeleniumAssignment_1.html");
+		if (driver.getTitle().equals("TECHNOCREDITS")) {
+			WebElement firstName = driver.findElement(By.id("first name"));
+			firstName.sendKeys("Mahesh");
+			driver.findElement(By.id("last name")).sendKeys("Kumavat");
+			driver.findElement(By.id("E-mail")).sendKeys("MaheshKumavat@Gmail.com");
+			driver.findElement(By.id("Company Name")).sendKeys("Citi");
+			WebElement genderMale = driver.findElement(By.id("male"));
+			if(genderMale.isSelected())
+				driver.findElement(By.id("female")).click();
+			else
+				genderMale.click();
+			driver.findElement(By.id("10+")).click();
+			driver.findElement(By.id("planguage")).sendKeys("Java,Python,VB");
+			WebElement java = driver.findElement(By.id("java"));
+			java.click();
+			driver.findElement(By.id("python")).click();
+			WebElement thoughtworks = driver.findElement(By.id("thoughtworks"));
+			thoughtworks.click();
+			driver.findElement(By.id("terms")).click();
+			driver.findElement(By.id("reset")).click();
+			if (firstName.getAttribute("value").equals("") && !thoughtworks.isSelected() && !java.isSelected()) {
+				System.out.println("Reset Successful");
+				driver.findElement(By.id("redirect")).click();
```

Comment:
Here you can write below steps out of if, because form reset and new page redirection does not have any relation.